library/globals.inc.php

```diff
@@ -1242,6 +1242,13 @@
       xl('Number of Appointments to display in the Patient Summary')
     ),
 
+    'recurrences_widget' => array(
+      xl('Recurrence Display Widget'),
```

Comment:
And move this setting above the following setting:
Past Appointment Display Widget